src/components/InputField/InputField.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import "./InputField.css"
+
+function InputField({ inputValue, onClearInput, onChangeInput }) {
+
+    const handleChange = (event) => {
+        onChangeInput(event.target.value)
+    }
+
+
+    return (
+        <input className='input-field' value={inputValue} type="text" onChange={handleChange} placeholder='Add task' />
```

Comment:
It's good practice to start each prop from new line if you have several props. It makes your code more readable.
Don't let your code grow to right, let it grow down instead.

Pros:
 - you can easily detect all properties
 - you don't need big monitor (usually 80-120 chars is a maximum line length)
 - you can clearly see where your tags start and where it ends

```
<input
  className='input-field'
  value={inputValue}
  type="text"
  onChange={handleChange}
  placeholder='Add task'
/>
```